app/controllers/YourSelfAssessmentChargesController.scala

```diff
@@ -66,8 +66,10 @@ class YourSelfAssessmentChargesController @Inject()(val authActions: AuthActions
 
       val hasOverdueCharges: Boolean = whatYouOweChargesList.chargesList.exists(_.isOverdue()(dateService))
       val hasAccruingInterestReviewAndReconcileCharges: Boolean = whatYouOweChargesList.chargesList.exists(_.isNotPaidAndNotOverduePoaReconciliationDebit()(dateService))
+      val hasChargesDueWithin30Days: Boolean = whatYouOweChargesList.chargesDueWithin30DaysList.nonEmpty
```

Comment:
should be fine to move this one on the level of WhatYouOweChargeList type, but really up to you if you keen to make this more concise